docs/src/pages/components/data-grid/columns/RenderExpandCellGrid.js

```diff
@@ -1,29 +1,11 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
+import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
 import Paper from '@mui/material/Paper';
 import Popper from '@mui/material/Popper';
-import { createStyles, makeStyles } from '@mui/styles';
 import { DataGrid } from '@mui/x-data-grid';
 
-const useStyles = makeStyles(() =>
```

Comment:
I would have just converted this to a styled component, but this works too. I usually made this decision but the length of the `sx` prop, if it doesn't fit in one line, I would use styled().